deploy/examples/ApplicationMonitoring.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: applicationmonitoring.integreatly.org/v1alpha1
 kind: ApplicationMonitoring
```

Comment:
FYI, I am planning to move the creation of the CR in here to installer with https://github.com/integr8ly/installation/pull/542
Because the installer role that creates the CR needs to use different namespaceSelectors and CR names for installing middleware monitoring and application metrics cases.